### DISCUSSION TITLE
issue

### DISCUSSION THREAD

**Discussion by ghost** [martinet101/ElevenClock#227] [General]
I finally rollbacked to windows 10 because i had issues with ElevenClock on windows 11
the problem was that when using the clock on second screen and watching a video on full screen 
the clock would appear over the fullscreened video and then when I try to click to remove the fullscreen it clicks on the clock and fucks everything up lol

so the issue is that the clock appears over fullscreened videos

**Comment by marticliment** [comment]
Do you have fullscreen hiding enabled in the setings window?

**Reply by ghost** [reply]
ok so that was a feature I didnt see I guess that fixes my issue, but imma still stick to windows 10 for now because windows 11 is too shitty.

i cant even right click my task bar to open task manager or click the volume to change the headset output

**Reply by marticliment** [reply]
Yes, i agree with you, Windows 11 should be better

**Reply by ghost** [reply]
thanks for the help tho, eleven cloc is a pretty good software I could see the seconds and even the day in my taskbar it was pretty lit
